src/test_typing_extensions.py

```diff
@@ -8098,6 +8100,563 @@ def test_pep_695_generics_with_future_annotations_nested_in_function(self):
             set(results.generic_func.__type_params__)
         )
 
+class TestEvaluateForwardRefs(BaseTestCase):
+    def test_evaluate_forward_refs(self):
+        from typing import ForwardRef  # noqa: F401 # needed for locals
```

Comment:
Do we really need this? Better to pass an explicit namespace instead of relying on the locals of this function.